Cargo.lock

```diff
@@ -3278,9 +3289,9 @@ dependencies = [
 
 [[package]]
 name = "regex-syntax"
-version = "0.6.26"
+version = "0.6.27"
```

Comment:
Need to rebase over master to undone some needless changes to Cargo.lock and submodules.

Reply:
Not resolved.